test/unittest_suite.py

```diff
@@ -25,6 +25,7 @@
         modules.append('zypper_plugins.test_package_upload')
         modules.append('zypper_plugins.test_tracer_upload')
         modules.append('test_katello.test_tracer')
+        modules.append('test_katello.test_deb_tracer')
```

Comment:
`zypper_enabled` implies Suse testing. I would not expect this here. Instead, if this is really a thing that needs to be tested on Python 2, I'd expect its own `try/except` block.

Reply:
It should not be tested on suse systems, I will remove this line.